layouts/fastly/helix.vcl

```diff
@@ -1781,23 +1785,25 @@ sub hlx_bereq {
   if (req.http.X-Request-Type != "Preflight") {
     unset bereq.http.X-Orig-Url;
     unset bereq.http.X-Orig-Host;
+    unset bereq.http.X-Owner;
+    unset bereq.http.X-Repo;
+    unset bereq.http.X-Ref;
+    # unset bereq.http.X-Repo-Root-Path;   // todo: should we remove this as well ?
```

Comment:
@trieloff this was not removed from the bereq before... is it important to the backends ?

Reply:
No, but for things like the cgi-bin it could be useful